src/status_im/extensions/core.cljs

```diff
@@ -272,6 +285,14 @@
                 {:permissions [:read]
                  :value       :extensions.chat.command/set-parameter-with-custom-params
                  :arguments   {:value :string :params :map}}
+                'chat.command/send-plain-text-message
+                {:permissions [:read]
+                 :value       :extensions.chat.command/send-plain-text-message
+                 :arguments   {:value :string}}
+                'chat.command/send
+                {:permissions [:read]
+                 :value       :extensions.chat.command/send
```

Comment:
Maybe `send-message` then?

Reply:
why message? it sends command )imo send is fine

Reply:
Well it really send a message no? I find it a bit confusing with `send-plain-text-message`

Reply:
hm, yes it's , i'll change it